<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FEFCFB;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>310</width>
     <height>622</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>310</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;</string>
   </property>
   <widget class="QPushButton" name="num_zero">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>572</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="point">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>572</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backspace">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>572</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resouce.qrc">
      <normaloff>:/rec/Icons/backspace.svg</normaloff>:/rec/Icons/backspace.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="equal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>572</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#dae4ed;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#d5dfe8;
}

QPushButton:pressed{
background-color:#c3ccd4;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_one">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>502</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_three">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>502</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_two">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>502</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>502</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#d2e4f4;
color:#353535;
border-radius:30px;
}
QPushButton:hover{
background-color:#cee0f0
}
QPushButton:pressed{
background-color:#bdcddb;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_four">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>432</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_six">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>432</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_five">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>432</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sub">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>432</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#d2e4f4;
color:#353535;
border-radius:30px;
}
QPushButton:hover{
background-color:#cee0f0
}
QPushButton:pressed{
background-color:#bdcddb;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_seven">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>362</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_nine">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>362</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_eight">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>362</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eef1f7;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#ebedf5;
}

QPushButton:pressed{
background-color:#d5d8de;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mult">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>362</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#d2e4f4;
color:#353535;
border-radius:30px;
}
QPushButton:hover{
background-color:#cee0f0
}
QPushButton:pressed{
background-color:#bdcddb;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>292</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#eadcfe;
color:#353535;
border-radius:30px;
}

QPushButton:hover{
background-color:#e6d9fa;
}

QPushButton:pressed{
background-color:#d5c8e6;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>292</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#d2e4f4;
color:#353535;
border-radius:30px;
}
QPushButton:hover{
background-color:#cee0f0
}
QPushButton:pressed{
background-color:#bdcddb;
}
</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brackets">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>292</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#d2e4f4;
color:#353535;
border-radius:30px;
}
QPushButton:hover{
background-color:#cee0f0
}
QPushButton:pressed{
background-color:#bdcddb;
}
</string>
    </property>
    <property name="text">
     <string>( )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="div">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>292</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#d2e4f4;
color:#353535;
border-radius:30px;
}
QPushButton:hover{
background-color:#cee0f0
}
QPushButton:pressed{
background-color:#bdcddb;
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:#dae4ed;
border-bottom-left-radius:25px;
border-bottom-right-radius:25px;
padding:10px;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>123</y>
      <width>280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:#ccd7de;
color:#353535;
border-radius:20px;
padding:10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ln">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>222</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>222</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>222</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>252</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>252</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>252</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pow">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_x">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>252</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>222</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_display_button">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>123</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:#b7c0c7;
color:#353535;
border-radius:20px;
}

QPushButton:hover{
background-color:#bdcddb;
}

QPushButton:pressed{
background-color:#bdcddb;
}
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exponent">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:#828282;
}

QPushButton:pressed{
border:none;
color:#353535;
}</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="graph"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="graph">
   <property name="text">
    <string>Нарисовать график</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+G</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
</ui>
